src/components/Header/Header.tsx

```diff
@@ -77,5 +74,4 @@ const Header = ({ nav, user }: Props) => {
     </header>
   );
 };
-
 export default Header;
```

Comment:
Why these deletions, in this file??? I don't think there should be any change in this file.